spec/builtin/ssl/ip_range_enum_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+require 'ronin/recon/builtin/net/ip_range_enum'
+
+describe Ronin::Recon::Net::IPRangeEnum do
+  describe "#process" do
+    context "when there are ip in range" do
+      let(:ip_range) { Ronin::Recon::Values::IPRange.new('192.168.0.1/2') }
+
+      it "must yield IP values" do
+        yielded_values = []
+
+        subject.process(ip_range) do |value|
+          yielded_values << value
+        end
+      end
+    end
+
+    context "when there is no ip in range" do
```

Comment:
This edge-case doesn't quite make sense. Even if the range was `1.2.3.4/32` it would still yield `1.2.3.4`.